main.md

```diff
@@ -136,8 +136,8 @@ This example shows a combined request asking for access to account information a
 [
    {  
       "type": "account_information",
-      "actions":
-        ["list_accounts", "read_balances", "read_transactions"],
+      "actions": ["list", "read"],
+	  "datatypes": ["accounts", "balance", "transactions"],
```

Comment:
I think I got it now. You want to have a means to express the datatypes at a certain API and actions on them. 

What is the meaning? Is the meaning the client is asking for read and list to all data types? This means if the client wants different sets of actions on the different data types, it needs to use multiple authz data objects?

If think the term "sub-resource" would be better since this typically also involves some path/navigation expression. If the type "transaction" would turn up in several places at the same API, you would most likely want to distinguish them.

Reply:
Yes, that's the idea. And it's exactly the desire to do different cross products of actions and types of data that I'm after, with each combination being its own object. "Sub-resource" would be fine, though I'd be worried about it getting confused with the external "resource" parameter.

Reply:
Consequently, I think we should go from "data" to "datatype" for the moment and see what shakes out from further conversation in the WG.

Reply:
wfm